pkgs/development/libraries/gettext/default.nix

```diff
@@ -52,7 +52,7 @@ stdenv.mkDerivation rec {
   strictDeps = true;
   nativeBuildInputs = [
     xz
-    xz.bin
+    (lib.getBin xz)
```

Comment:
```suggestion
```
I've run diffoscope on a gettext build with and without xz and booth are equal. I think we can just drop xz entirely from here.

and we can also drop the ``"--with-xz"`` arg in line 31.